synapseclient/annotations.py

```diff
@@ -335,6 +341,11 @@ def from_synapse_annotations(raw_annotations: typing.Dict[str, typing.Any]) -> A
     return annos
 
 
+def check_annotations_changed(bundle_annotations, new_annotations):
+    transferred_annotations = _convert_to_annotations_list(new_annotations)
```

Comment:
Maybe rename this variable? transferred seems an odd name? Maybe converted_annos?